content/projects/understanding-loops/_index.md

```diff
@@ -65,6 +65,26 @@ Please note that one person can have multiple baskets. If you look at the data,
 
 ## Write some functionality
 
+### Important notes for Javascript solutions
+
+In Javascript it is best practice to use destructuring when passing arguments to a function. You can learn about destructuring here: [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
+
+Please make sure you use destructuring for all your functions. 
+
+```
+getCustomerBaskets(cart, "sine@umuzi.org"); // DON'T DO THIS
+getCustomerBaskets("sine@umuzi.org", cart); // DON'T DO THIS EITHER
+
+// The problem with the above code is that if you get the order of the parameters wrong, then things will break. Your functions should work like this instead:
+
+function getCustomerBaskets({ cart, email }) {...}
+
+const email = "sine@umuzi.org";
+const cart = someJsonArray
```

Comment:
Shouldn't this represent the actual object that stores the data the learners are expected to work with? Instead of these being 2 separate constants.